keras/layers/convolutional.py

```diff
@@ -220,6 +215,8 @@ class Convolution2D(Layer):
         border_mode: 'valid' or 'same'.
         subsample: tuple of length 2. Factor by which to subsample output.
             Also called strides elsewhere.
+        filter_dilation: tuple of length 2. Factor for kernel dilation.
+            Used by dilated convolution(a.k.a. convolution with holes or atrous convolution).
```

Comment:
the one you suggested is good, but I think this one has a better pictorial representation and they train the model from scratch using dilated convs instead of fine tuning VGG https://arxiv.org/pdf/1511.07122v3.pdf what do you think?